templates/button.php

```diff
@@ -0,0 +1,15 @@
+<?php
+/**
+ * Default template for Fieldmanager_Button
+ *
+ * @package Fieldmanager
+ */
+?>
+<input
+	class="fm-button-element button-secondary <?php echo esc_attr( $this->field_class ); ?>"
+	type="button"
+	name="<?php echo esc_attr( $this->get_form_name() ); ?>"
+	id="<?php echo esc_attr( $this->get_element_id() ); ?>"
+	value="<?php echo esc_attr( $this->options['button_text'] ); ?>"
```

Comment:
This isn't the way `options` is used, that field is for e.g. `<select>` options. Instead, `Fieldmanager_Button` should get a public property, `$button_text`, and then it would be set like any other field argument

Reply:
Ah, that makes sense. 👍